Yosemite/Yosemite/PointOfSale/POSProductProvider.swift

```diff
@@ -51,11 +51,12 @@ final class POSProductProvider {
 
         // 2. Map result to POSProduct and populate the output
         return loadedProducts.map { product in
+            // TODO: Decorate the price with currency formatting
             POSProduct(itemID: UUID(),
                        productID: product.productID,
                        name: product.name,
                        price: product.price,
-                       currencySettings: currencySettings)
+                       formattedPrice: "$\(product.price)")
```

Comment:
just checking, do you plan to create the price with currencySettings in this or a separate PR?

Reply:
I'm leaving it for a separate PR, since we plan to remove the currency settings dependency duplication on https://github.com/woocommerce/woocommerce-ios/issues/12918 . It may also be a good place to start simple adding unit test for the models and be sure currency and others are passed correctly.